middleware/backend/tests/debug_test.go

```diff
@@ -1,12 +1,12 @@
 // +build etcd
 
-package etcd
+package btests
```

Comment:
I had an issue with recursive imports (etcd importing backend, backend tests importing etcd).  I fixed it by moving it to another package. If you know a better way of doing it/naming, feel free to comment.